nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_Fault/reference/outputs/timeLine/timeline.log

```diff
@@ -4,10 +4,16 @@
 1 | GEN____1_SM | Voltage regulator : upper limit reached
 1 | GEN____2_SM | Voltage regulator : upper limit reached
 1 | GEN____6_SM | Voltage regulator : upper limit reached
+1 | GEN____1_SM | Voltage regulator : upper limit reached
+1 | GEN____2_SM | Voltage regulator : upper limit reached
+1 | GEN____6_SM | Voltage regulator : upper limit reached
```

Comment:
This is not really a comment demanding corrections but so the behavior is that a new event is added to the timeline as soon as it is not equal to the last event. It ends up in similar events being displayed as in this example because there are other events in between.

Do you already have in mind an idea to deal with this?

Reply:
This is a topic that I wanted to review with you. Unfortunately this is a behavior inherent to the way we deal with events. I don't see any easy way to determine on the fly that an event was already dumped and has no further effect or if it adds some information to the timeline. For example : the generator hits its voltage upper limit, then it goes down, and then another event makes it hit it again => do we want to display the message twice? We can freeze this MR if you want to discuss it further before merging.